docker-compose.yml

```diff
@@ -2,24 +2,50 @@ version: "3"
 services:
   mysql:
     image: mysql:5.7
+    restart: always
     environment:
       MYSQL_ROOT_PASSWORD: password
     volumes:
       - ./docker:/docker-entrypoint-initdb.d
-    ports:
-      - "3306:3306"
-    healthcheck:
-      test: ["CMD", "mysqladmin", "ping"]
-      interval: 10s
-      timeout: 3s
-      retries: 10
 
   redis:
     image: redis
+    restart: always
+
+  website:
+    build: .
+    depends_on:
+      - mysql
+      - redis
+    restart: always
+    environment:
+      MYSQL_HOST: mysql
+      MYSQL_SOCKET:
+      MYSQL_PORT: 3306
+      MYSQL_USERNAME: root
+      MYSQL_PASSWORD: password
+      REDIS_HOST: redis
+    ports:
+      - 3000:3000
```

Comment:
```suggestion
      - 5000:5000
```

Reply:
I've changed this to `5000:3000`, to keep the website internally at the default port. This is usually how this is done.